modules/sync/protocolbuffers/gofeatures/cas/66220fa6f78c2ff242ac897567ed4255f1bd5f76bf5e252edc2d72945e75504e73da2dfe9dce502d6e522a0242d2c3a58482836bfc0463f2a460fea985f99ccf

```diff
@@ -0,0 +1,2 @@
+version: v1
+name: buf.build/protocolbuffers/gofeatures
```

Comment:
Should this have a dependency on protocolbuffers/wellknowntypes? If so does it need to pin a version of the WKTs in the buf.yaml file?

Reply:
I think this should depend on WKT - without it I got the following error running the fetch script locally (with buf v1.31.0):
```
google/protobuf/go_features.proto:17:28:extension pb.go: tag 1002 is not in valid range for extended type google.protobuf.FeatureSet
```

Reply:
Hmm, unclear. The wellknowntypes never require an explicit dep since `buf` bundles them. They are only needed as a dependency to explicitly pin a particular version.

In this case, this version (1.34.0) requires v25.0 or higher of the wellknowntypes. And the _next_ version will require v27.0 or higher. But will that be weird to pin it? Would that mean that all modules that _use_ the gofeatures end up getting "stuck" on whatever version it pins? (I.e. they wouldn't end up using newer wellknowntypes, even if using a newer version of the CLI that bundles a newer version of them.)

Reply:
I agree with Philip here, let's make it a dependency

Reply:
What's our stance on the version to pin: should it be the _minimum_ version required (v25.0)? Or should we keep it recent (e.g. v26.1)?

I'm going to guess the minimum one since it doesn't look like we have anything automated to update it as newer versions of deps are released.

Reply:
I think we can leave it unpinned if we want - it will just point at the latest WKT version when we sync on the user's system. That also means that we don't have to remember to bump the pinned version in the future.

Reply:
> it will just point at the latest WKT version when we sync on the user's system.

But if the user has already pinned the WKT module, but an older version, it will fail to build, right? I'm also wondering if we should do something different than leaving it unpinned in the next version (which will require a new v27+ of the wellknowntypes).

In fact, that makes me wonder if maybe we should hold off on trying to merge this until we have a v27 of wellknowntypes and a v1.35 or v1.34.1 (whatever they number it) for protobuf-go and not bother trying to sync the current v1.34.0...

Reply:
Now that v27.0 of protocolbuffers/wellknowntypes is available, I've updated this so that it will start with v1.34.1 (which requires v27 of descriptor.proto). Updated the config in this PR.

I'll now start going through the checklist for new modules.